### ISSUE TITLE
Set window for timeseries plot? #Question

### ISSUE THREAD

**Issue by JadeMaveric** [altair-viz/altair#1593]
I'm plotting points for a temperature over course of a day. The readings I have span over 6 hours, but the samples were taken 100ms apart.

I'm using the following bit of code to generate the chart
```
chart = alt.Chart(rooms).mark_point().encode(
    x = 'timestamp:T',
    y= 'sensor',
    color = 'location:N'
).interactive()
```
This of course displays the data over the entire 6 hours. Whereas I would like to see the variations over 10 second windows. I've tried searching but haven't been able to figure out how to set the window. And zooming in from hours to seconds in a pain. Is there anyway to set it so that the plot is zoomed in when it first renders?

**Comment by jakevdp** [comment]
This is not possible in the current release of Altair, but initialization of scale-bound selections was added in vega-lite here: https://github.com/vega/vega-lite/pull/5352

This will be available in Altair version 4.0, when it is released.

**Comment by joelostblom** [comment]
Closing since altair 4 (and 5 :tada: ) has since been released.
